test/system/505-networking-pasta.bats

```diff
@@ -335,14 +335,12 @@ function teardown() {
 }
 
 @test "podman networking with pasta(1) - Local forwarder, IPv4" {
-    skip "FIXME: #17074: some pasta dns problem"
     skip_if_no_ipv4 "IPv4 not routable on the host"
 
     run_podman run --dns 198.51.100.1 \
-        --net=pasta:--dns-forward,198.51.100.1 $IMAGE nslookup 127.0.0.1
+        --net=pasta:--dns-forward,198.51.100.1 $IMAGE nslookup 127.0.0.1 || :
```

Comment:
I see your commit message but why try to handle different behaviour? Wouldn't it be better to just nslookup google.com? We already do this in other tests so we know it works in all environments where the tests are run at the moment.

Also if a test accepts a unknown exit code write it as `run_podman ? arguments...`

Reply:
> I see your commit message but why try to handle different behaviour? Wouldn't it be better to just nslookup google.com? We already do this in other tests so we know it works in all environments where the tests are run at the moment.

Two reasons, but not necessarily very practical:
- I'm not really fond of hardcoding in tests of publicly available software the domain name of a given company. I'm also not sure that google.com can be universally resolved. I kind of expect DNS censorship to be implemented with DNS poisoning rather than just dropping queries, but I wouldn't be so sure
- the host resolver might be temporarily unable to contact an upstream DNS, but this shouldn't affect this test in particular.

If you have a strong preference toward that, I'll change it, it's probably theoretical problems I'm posing.

> Also if a test accepts a unknown exit code write it as run_podman ? arguments...

Oh, I didn't know that, I'll change it.

Reply:
You can keep your version, the test does its job. Not depending an a particular name is properly better but as said above we already use it so one more wouldn't make a difference.